objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
```

Comment:
Good work on this but still it's not doing what it is supposed to do.
Follow the steps below to solve it.
1. Create an empty object outside of the for loop. I explained to you how to create an empty object [here](https://github.com/SeR4h/Andelawork/pull/1#discussion_r348401151)
2. Loop through the numbers from 1 to 15.
3. For each number, Calculate its square
Add the number as the key to the object you created outside the loop and its square as the value. I explained to you how to do it in [this conversation](https://github.com/SeR4h/Andelawork/pull/1#discussion_r348401151) 
4. When the loop is  done (below the loop) you can log the object